docs/en/integrations/language-clients/java/client-v1.md

```diff
@@ -14,7 +14,9 @@ import CodeBlock from '@theme/CodeBlock';
 
 Java client library to communicate with a DB server thru its protocols. Current implementation supports only [HTTP interface](/docs/en/interfaces/http). The library provides own API to send requests to a server.  
 
-*Note*: this component will be deprecated soon. 
+:::warning Deprecation
+This library will be depreceted on `February, 28, 2025`. It means no releases after and support. 
```

Comment:
spelling `depreceted` -> `deprecated`. `It means no releases after and support. ` -> `This means no further releases and support issues will not be addressed. `